app/models/planning_application.rb

```diff
@@ -704,7 +705,7 @@ def open_post_validation_requests
   end
 
   def closed_pre_validation_requests
-    validation_requests.closed - validation_requests.closed.post_validation
+    validation_requests.where.not(type: "time_extension_validation_request").closed - validation_requests.closed.post_validation
```

Comment:
We're using this condition in a few places (the controller too) — should we make it a scope or something perhaps?